bin/load-runner.js

```diff
@@ -46,19 +52,24 @@ var args = require('optimist')
     'demand': true,
     'describe': 'Script to execute'
   })
+  .options('seed', {
```

Comment:
So what I was saying in the comment below, is that you can give this option a property like `type: number` (unfortunately there's no `int` type).

I also suggested removing the `'default': '-1'` if you go for the suggested change.